linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_remote_access_monitoring/tests/differnt_files.pass.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# platform = multi_platform_all
+
+declare -A REMOTE_METHODS=( ['auth.*']='^.*auth\.\*.*$' ['authpriv.*']='^.*authpriv\.\*.*$' ['daemon.*']='^.*daemon\.\*.*$' )
+RSYSLOG_CONF='/etc/rsyslog.conf'
+RSYSLOG_D_FOLDER='/etc/rsyslog.d'
+RSYSLOG_D_FILES='/etc/rsyslog.d/*'
+
+
+# clean up .d conf files (if applicable)
+if [[ -d ${RSYSLOG_D_FOLDER} ]]; then
+	for rsyslog_d_file in ${RSYSLOG_D_FILES}
+	do
+		for K in ${!REMOTE_METHODS[@]}
+		do
+			if grep -q "${REMOTE_METHODS[$K]}" ${rsyslog_d_file}; then
+				sed -i "/${REMOTE_METHODS[$K]}/d" ${rsyslog_d_file}
+			fi
+		done
+	done
+fi
+
+if [[ ! -f /etc/rsyslog.conf ]]; then
+	# Something is not right, create the file
+	touch /etc/rsyslog.conf
+fi
+
+echo "auth.*,authpriv.* /var/log/secure" >> $RSYSLOG_CONF
```

Comment:
Should there be a comma or a semicolon between the asterisk and a? The rule description has a semicolon and here you have a comma.